fortran/test/GildedRose_unity_test.f90

```diff
@@ -0,0 +1,20 @@
+
+
+
+
+
+
+program GildedRose_texttest
+  use GildedRose
+  implicit none
+  type(Item) :: it
+
+  call init_item(it, "foo", 10, 20)
```

Comment:
I wonder if there should be a call here to update_quality before the assertion?

Reply:
You are right, I missed that. Not quite sure what this test is doing but that is kind of the point, right. ;-)
Thanks for maintaining this valuable resource.